agent/src/containeroptions/dynamiccontaineroptions.cpp

```diff
@@ -40,4 +40,24 @@ bool DynamicContainerOptions::enableWriteBuffer() const
     return m_enableWriteBuffer;
 }
 
+void DynamicContainerOptions::setEnableTemporaryFileSystemWriteBuffers(bool enabled)
+{
+    m_enableTemporaryFileSystemWriteBuffers = enabled;
+}
+
+bool DynamicContainerOptions::enableTemporaryFileSystemWriteBuffers() const
```

Comment:
I understand your point ;) . We do have a lot of Qt coding style guidelines which I think are reasonable, and this naming does follow those ... or well, they don't mention it really, but their code is the same as this at least ;). Not sure what others feel like? @tobsan @breakreturn ?